docs/user_guide/labels_and_annotations_in_yunikorn.md

```diff
@@ -25,19 +25,21 @@ under the License.
-|-----------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `applicationId`                   | Associates this pod with an application.                                                                                                                |
-| `queue`                           | Selects the YuniKorn queue this application should be scheduled in. This may be ignored if a placement policy is in effect.                             |
-| `spark-app-selector`              | Alternative method of specifying `applicationId` used by Spark Operator if the label `applicationId` and annotation `yunikorn.apache.org/app-id` unset. |
+Label values should comply with [Kubernetes Syntax and character set](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set).
+
+| Name                              | Description                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+|-----------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `applicationId`                   | Associates this pod with an application.                                                                                                                                                                                                                                                                                                                                                                                                                |
+| `queue`                           | Selects the YuniKorn queue this application should be scheduled in. Queue name should comply with [Kubernetes Syntax and character set](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set) and also with [Partition and Queue Configuration](queue_config#queues). If Queue name doesn't comply, Annotations could be used as described below. This may be ignored if a placement policy is in effect. |
```

Comment:
nit grammar/spelling: "if the queue doesn't comply, annotations could be used as described below"